pkg/apis/hub/v1alpha1/ai_service.go

```diff
@@ -50,9 +50,15 @@ type AIServiceSpec struct {
 
 // +k8s:deepcopy-gen=true
 
+type Secret struct {
+	SecretName string `json:"secretName"`
+}
+
+// +k8s:deepcopy-gen=true
+
 // Anthropic configures Anthropic backend.
 type Anthropic struct {
-	Token  string  `json:"token"`
+	Token  *Secret `json:"token,omitempty"`
```

Comment:
Out of curiosity, is there a reason for AIServices to have a distinct `Token` or `APIKey` option?